app/src/main.c

```diff
@@ -210,12 +217,28 @@ int main(void) {
         return 0;
     }
 
+    uint8_t main_imu_trigger = 0;
```

Comment:
Can we use a clearer name and a boolean here? Something like `bool main_imu_using_triggers/interrupts` or similar.